### DISCUSSION TITLE
brightness control not working!

### DISCUSSION THREAD

**Discussion by mursalatul** [i3/i3#5551] [Q&A] [answered]
## I'm submitting a…

<pre>
[x] Bug
[ ] Feature Request
[ ] Documentation Request
[ ] Other (Please describe in detail)
</pre>

## Current Behavior

brightness is not changing when i am pressing the XF86MonBrightnessUp/XF86MonBrightnessDown button (OS: garuda i3).

## Expected Behavior
Brightness should change as i press the dedicated buttons.

## Environment
OS: 
Device: dell inspiron 5558
Output of `i3 --moreversion 2>&-`:
<pre>
i3 version: 4.22
</pre>

<details><summary>Config file</summary><pre>
[](https://pastebin.com/Hs4qdiNt)
</pre>
</details>

<pre>
Logfile URL:
</pre>

<pre>
- Linux Distribution & Version: Garuda i3 
- Are you using a compositor (e.g., xcompmgr or compton): No
</pre>

**Comment by orestisfl** [comment]
i3 does nothing directly to control brightness. You can bind these keys to appropriate actions. The arch wiki includes a nice article for your options: https://wiki.archlinux.org/title/Backlight

[light](https://github.com/haikarainen/light) is a very popular choice but unfortunately the project is now considered "orphaned"

**Reply by orestisfl** [reply]
[brightnessctl](https://github.com/Hummer12007/brightnessctl) seems to be a good alternative

**Reply by mursalatul** [reply, marked as answer]
i found a solution.
replace
```
bindsym XF86MonBrightnessUp exec "xbacklight -inc 10; notify-send 'brightness up'"
bindsym XF86MonBrightnessDown exec "xbacklight -dec 10; notify-send 'brightness down'"
```
with
```
bindsym XF86MonBrightnessUp exec "xbacklight -inc 10; notify-send 'brightness up'; brightnessctl set 5%+"
bindsym XF86MonBrightnessDown exec "xbacklight -dec 10; notify-send 'brightness down'; brightnessctl set 5%- --min=3%"
```
now you can change the brightness of your device.
